packages/react/src/components/Flex/FlexItem.tsx

```diff
@@ -78,7 +78,7 @@ class FlexItem extends UIComponent<Extendable<FlexItemProps>> {
       })
     }
 
-    return applyStyles(React.Children.only(children), styles, classes)
+    return children ? applyStyles(React.Children.only(children), styles, classes) : children
```

Comment:
```suggestion
    return _.isNil(children) ? applyStyles(React.Children.only(children), styles, classes) : null
```

May be this will be more clear?